helix-front/client/app/shared/node-viewer/node-viewer.component.ts

```diff
@@ -254,29 +298,24 @@ export class NodeViewerComponent implements OnInit {
       case 'list':
         if (key) {
           const entry = _.find(this.node.listFields, { name: key });
-          //         Property 'value' does not exist on type 'number | ListFieldObject | ((searchElement: ListFieldObject, fromIndex?: number) => boolean) | ((...'.
-          // Property 'value' does not exist on type 'number'.ts(2339)
-          // @ts-ignore
+
```

Comment:
Good news, we don't need this anymore.  This code no longer produces an error, with the more modern version of TypeScript we are now using.